### ISSUE TITLE
Module not found: ./../../process/browser.js

### ISSUE THREAD

**Issue by jamesgis** [crackernutter/arcgis-webpack-vue-calcite#1]
for example 
**import AreaMeasurement3D from "esri/widgets/AreaMeasurement3D";**

chrome console error below:
Uncaught Error: Module not found: ./../../process/browser.js
    at findModule (app.js:985)
    at contextRequire (app.js:1011)
    at req (app.js:917)
    at eval (geometryEngine.js:1309)
    at Object../node_modules/arcgis-js-api/geometry/geometryEngine.js (app.js:5609)
    at __webpack_require__ (app.js:767)
    at fn (app.js:130)
    at eval (measurementUtils.js?3989:25)
    at Object../node_modules/arcgis-js-api/views/3d/interactive/measurementTools/support/measurementUtils.js (app.js:10658)
    at __webpack_require__ (app.js:767)

**Comment by crackernutter** [comment]
@jamesgis In vue.config.js, in configureWebpack.node you might want to try commenting out process: false and/or fs:"empty".  The amount of experimenting I had to do to get this all to work (and just in Chrome nonetheless) was a lot, and I'm still not even sure it's foolproof - I'm sure there are scenarios which might break the whole thing.  That being said, I have resolved other issues with by editing the node object in vue.config.js. 
Also sorry for the late reply!
